blogs/ecosystems-and-partners/zyphra/README.md

```diff
@@ -0,0 +1,125 @@
+---
+blogpost: true
+date: 10 December 2024
+blog_title: "Zyphra Introduces Frontier Training Kernels for Transformers and SSMs on AMD Instinct MI300X Accelerators"
+thumbnail: 'amd_zyphra.jpg'
+author: Quentin Anthony
+tags: LLM, AI/ML, GenAI, PyTorch
+category: Ecosystems and Partners
+language: English
+myst:
+  html_meta:
+    "description lang=en": "This blog shows Zyphra's new training kernels for transformers and hybrid models on AMD Instinct MI300X accelerators, surpassing the H100s performance"
+    "keywords": "Mamba, PyTorch, S4, S6, Mamba2, Transformer, Flash Attention, Optimization, Hardware-aware, Transformer, Attention, ROCm, Mi210, MI250, MI300, AI/ML, Generative AI"
+    "property=og:locale": "en_US"
+---
+
+
+# Training Transformers and Hybrid models on AMD Instinct MI300X Accelerators
+
+## Harnessing the MI300 Hardware Specs
```

Comment:
No section title here, please. The post just starts with the brief intro text, with no section title (this section title goes in, but only a little further, please see below). Thanks!
```suggestion
```